jobs/uaa/spec

```diff
@@ -744,6 +744,16 @@ properties:
     description: "how long the results of a preflight request is cached"
     default: ~
 
+  # Content Security Policy (csp) settings
+  uaa.csp.script-src:
+    description: "Overrides the default script-src CSP header value of 'self'. Set this to allow scripts to be loaded from sources besides UAA. Because this overrides (not appends to) the value, be sure to include an entry for 'self' so scripts from UAA are allowed."
```

Comment:
I would like to see something that explains that this is dangerous. That especially if you want to use `unsafe-inline` you are opening yourself up to javascript code injection attacks. Like described [here](https://content-security-policy.com/unsafe-inline/)

Reply:
yes but as always, if you want overwrite defaults that can be dangerous, e.g. skipVerify in TLS
The test is from https://github.com/cloudfoundry/uaa-release/pull/514 but if you have an example we can add